python/ray/autoscaler/_private/_azure/azure-config-template.json

```diff
@@ -2,42 +2,54 @@
     "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
     "contentVersion": "1.0.0.0",
     "parameters": {
+        "uniqueId": {
+            "type": "string",
+            "metadata": {
+                "description": "Unique string for resources in the same cluster."
```

Comment:
Can you mention why we'd want to set this, and what it defaults to?

Reply:
will do, though there is no default here, that is set prior to exercising this template.